[T7][W09-C2]

antonong94 opened the pull request.

damithc commented on config.json:
```diff
@@ -0,0 +1,7 @@
+  "userPrefsFilePath" : "preferences.json",
+  "taskListFilePath" : "data/tasklist.xml",
+  "taskListName" : "MyTaskList"
+}
```
This file shouldn't be in the repo.

damithc commented on docs/AboutUs.md:
```diff
@@ -4,49 +4,33 @@ We are a team based in the [School of Computing, National University of Singapor
+<img src="images/DuanYiChen.jpg" width="150"><br>
 Role: Developer <br>  
-Responsibilities: UI
+Responsibilities: Testing
```
Also mention which component you are in charge of

damithc commented on docs/UserGuide.md:
```diff
@@ -39,96 +39,166 @@
-#### Adding a person: `add`
-Adds a person to the address book<br>
-Format: `add NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...` 
+#### Adding a floating task: `add`
```
Is the `floating task` the best name to use here? It's not a commonly used term.

damithc commented on docs/UserGuide.md:
```diff
@@ -39,96 +39,166 @@
+* `add Homework tag/CS1231`
+
+#### Adding a task with deadline: �add�
+Format: `add TASK_NAME end/DATE TIME [tag/TAG]...`
```
Consider supporting shorter forms of keywords. Advanced users should be able to do the same actions with fewer key strokes.

damithc replied:
`end` doesn't sound like a deadline.

damithc commented on docs/UserGuide.md:
```diff
@@ -39,96 +39,166 @@
+
+Examples:
+* `add Homework end/2409 1800 tag/CS1231`
+* `add Homework end/24 sep 6pm tag/CS1231`
```
You need to specify which date/time formats you support

damithc commented on docs/UserGuide.md:
```diff
@@ -39,96 +39,166 @@
+Format: `add TASK_NAME start/DATE TIME end/DATE TIME [tag/TAG]`
+
+> Words in `UPPER_CASE` are the parameters, items in `SQUARE_BRACKETS` are optional, 
+> items with `...` after them can have multiple instances. Order of parameters are fixed. 
```
Best not to repeat the same info in many places.

damithc commented on docs/UserGuide.md:
```diff
@@ -39,96 +39,166 @@
+Examples:
+* `Delete 212`
+
+#### Block out tasks : �block�
```
Need to explain what you mean by 'block'

damithc commented on docs/UserGuide.md:
```diff
@@ -39,96 +39,166 @@
-#### Finding all persons containing any keyword in their name: `find`
-Finds persons whose names contain any of the given keywords.<br>
-Format: `find KEYWORD [MORE_KEYWORDS]`
+> Maximum 3 redo
```
Why 3? sounds like an arbitrary restriction :-p

damithc commented on docs/UserGuide.md:
```diff
@@ -39,96 +39,166 @@
-Selects the person identified by the index number used in the last person listing.<br>
-Format: `select INDEX`
+#### Change directory: �Change directory�
+Format: cd FILE_PATH
```
What  if I want to save in the same directory, but using a different name?

damithc commented:
Some comments added. Please close the PR after reading comments.

Yichen-D committed:
Merge pull request #120 from CS2103AUG2016-W09-C2/more-testcases

Refactor and increase test coverage for UniqueTaskList

Yichen-D committed:
Fix undo/redo display. Fix taskmaster copier failure.
Allow more flexible editing regarding recurring format.

Yichen-D committed:
Change ageda css color theme

Yichen-D committed:
Minor change

Yichen-D committed:
Add tentative guitest for view command, fix parser bug when adding unparsable time slot.
Modify logic test to ensure undo/redo do works
After undo restore last shown filter component list ongoing

Yichen-D committed:
Remove used imports

Yichen-D committed:
Update DeveloperGuide.md

Yichen-D committed:
Update UI component

Yichen-D committed:
Minor changes.

Yichen-D committed:
Merge pull request #121 from CS2103AUG2016-W09-C2/fix-undo-redo

Fix undo redo

brandonyeoxg committed:
Add documentation.
Update diagrams.

brandonyeoxg committed:
Update diagram.

brandonyeoxg committed:
Remove uncessary pictures.

brandonyeoxg committed:
Update diagram.

billstark committed:
add author

Yichen-D committed:
Upload images for undo redo

billstark committed:
Merge remote-tracking branch 'origin/development' into change-documentation

brandonyeoxg committed:
Refactor RecurringTaskManager, the sentences were too long.

Yichen-D committed:
Update DeveloperGuide.md

brandonyeoxg committed:
Update developer guide to include pictures of code snippets in the testing section.

Yichen-D committed:
Add files via upload

Yichen-D committed:
Delete URManager.jpg

brandonyeoxg committed:
Fix image broken link.

Yichen-D committed:
Add files via upload

Yichen-D committed:
Update DeveloperGuide.md

brandonyeoxg committed:
Fix formatting of testing section.

Yichen-D committed:
Update DeveloperGuide.md

brandonyeoxg committed:
Merge pull request #124 from CS2103AUG2016-W09-C2/add-documentation

Add documentation

brandonyeoxg committed:
Update README to reflect new covereage branch.

brandonyeoxg committed:
Fix user guide from a merge conflict.

billstark committed:
update user guide

brandonyeoxg committed:
Remove coveralls in hopes that coverall will reupdate.

brandonyeoxg committed:
Add back coveralls. Hopefully it updates.

brandonyeoxg committed:
no message

brandonyeoxg committed:
Reupdate coveralls again.

billstark committed:
Merge branch 'master' of https://github.com/CS2103AUG2016-W09-C2/main

# Conflicts:
#	docs/DeveloperGuide.md
#	docs/images/SD_Delete_Floating_Interaction.png
#	docs/images/SD_Delete_Floating_Task.png
#	docs/images/UI Component.png

billstark committed:
Merge branch 'master' of https://github.com/CS2103AUG2016-W09-C2/main

# Conflicts:
#	README.md

billstark committed:
Merge remote-tracking branch 'origin/master' into change-documentation

# Conflicts:
#	docs/UserGuide.md

brandonyeoxg committed:
Edit User Guide add command.

Yichen-D committed:
Merge pull request #128 from CS2103AUG2016-W09-C2/change-documentation

Change documentation

billstark committed:
Merge branch 'master' of https://github.com/CS2103AUG2016-W09-C2/main

# Conflicts:
#	docs/UserGuide.md

billstark committed:
trying to fix display

Yichen-D committed:
Upload pics for ug

brandonyeoxg committed:
Merge pull request #129 from CS2103AUG2016-W09-C2/add-ug-documentation

Edit User Guide add command.

Yichen-D committed:
Update README.md

Yichen-D committed:
Update UserGuide.md

brandonyeoxg committed:
Add images for user guide.

Yichen-D committed:
Add files via upload

brandonyeoxg committed:
Merge pull request #130 from CS2103AUG2016-W09-C2/fix-user-guide

Fix user guide

Yichen-D committed:
Update UserGuide.md

Yichen-D committed:
Update AutoCompleteTextField.java

brandonyeoxg committed:
Add collate for DG and UG

Yichen-D committed:
Update README.md

brandonyeoxg committed:
Merge pull request #132 from CS2103AUG2016-W09-C2/add-ug-documentation

Add images for user guide.

Yichen-D committed:
Merge pull request #131 from CS2103AUG2016-W09-C2/Edit-documentation

Update README.md

Yichen-D committed:
Update UserGuide.md

Yichen-D committed:
Update UserGuide.md

antonong94 committed:
no message

antonong94 committed:
add collate

brandonyeoxg committed:
Update UserGuide.md

brandonyeoxg committed:
Update UserGuide.md

Fix formatting.

Yichen-D committed:
Update UserGuide.md

antonong94 committed:
update README.md

Yichen-D committed:
Update DeveloperGuide.md

Yichen-D committed:
Update UserGuide.md

Yichen-D committed:
Update UserGuide.md

antonong94 committed:
Merge pull request #134 from CS2103AUG2016-W09-C2/refractor_and_collate

Refactor and collate

antonong94 committed:
Merge branch 'master' of https://github.com/CS2103AUG2016-W09-C2/main

antonong94 committed:
Update Userguide.md and collate

Yichen-D committed:
collate

Yichen-D committed:
Rename demolist.xml to tasklist.xml

demo file for v0.4, store it in ./data\

antonong94 committed:
update UserGuide.md

antonong94 committed:
update UserGuide.md

antonong94 committed:
update AboutUs.md

antonong94 committed:
Update AboutUs.md

antonong94 committed:
Update AboutUs.md

Yichen-D committed:
recover some screwed up collate

Yichen-D committed:
Update UserGuide.md

Yichen-D committed:
Update AboutUs.md

Yichen-D committed:
Update UserGuide.md

Yichen-D committed:
Update UserGuide.md

Yichen-D committed:
Update AboutUs.md

Yichen-D committed:
Update DeveloperGuide.md

Yichen-D committed:
Update AboutUs.md

Yichen-D committed:
Update AboutUs.md

Yichen-D committed:
MUTATE TASK LATER SELECT

Yichen-D committed:
minor

Yichen-D committed:
Merge pull request #140 from CS2103AUG2016-W09-C2/enhance-select

Enhance select

Yichen-D committed:
Remove redundant methods in ui part

Yichen-D committed:
Update README.md

Yichen-D committed:
Improve select related

Yichen-D committed:
Merge pull request #144 from CS2103AUG2016-W09-C2/improve-ui

Improve select related

Yichen-D committed:
Revert "Improve select related"

Yichen-D committed:
Merge pull request #145 from CS2103AUG2016-W09-C2/revert-144-improve-ui

Revert "Improve select related"

brandonyeoxg committed:
Refactor code based on feedback.

brandonyeoxg committed:
Fix test method naming to follow coding standard.

brandonyeoxg committed:
Change how RecurringTaskManagerTest test for assertions.

brandonyeoxg committed:
Fix bug where startDate = null raises an assertion.

damithc commented on collated/main/A0135782Y.md:
```diff
@@ -0,0 +1,1063 @@
+        TaskComponent newAppendedDate = new TaskComponent((Task) task, editedStartDate, editedEndDate);
+        task.appendRecurringDate(newAppendedDate);
+        repeatingTasks.appendTaskComponent(newAppendedDate);
+    }
```
Some methods look very similar. e.g. this method and the previous method. Try to abstract out similarities to reduce duplication.

damithc commented on collated/main/A0135782Y.md:
```diff
@@ -0,0 +1,1063 @@
+    	Task task = new Task(name, tags);
+    	
+    	if(isArchived){
+    		task.setTaskType(TaskType.COMPLETED);
```
Looks like the indentation is wrong.

damithc commented on collated/main/A0147967J.md:
```diff
@@ -0,0 +1,1224 @@
+        	indicateAttemptToExecuteFailedCommand();
+        	urManager.popFromUndoQueue();
+        	return new CommandResult(MESSAGE_TIMESLOT_OCCUPIED);
+		} catch (DuplicateTaskException e) {
```
Wrong indentation

damithc commented on collated/main/A0147967J.md:
```diff
@@ -0,0 +1,1224 @@
+			if(!isIgnored(command)){
+				if(undoQueue.size() == MAX_TIMES) undoQueue.removeFirst();
+				undoQueue.addLast(new Context(model, command));
+				redoQueue.clear();
```
Minimize arrowhead anti pattern https://lostechies.com/chrismissal/2009/05/27/anti-patterns-and-worst-practices-the-arrowhead-anti-pattern/

damithc commented on collated/main/A0147967J.md:
```diff
@@ -0,0 +1,1224 @@
+        	}
+        }
+        return false;
+    }
```
This method can use some SLAP

damithc commented on collated/main/A0147967Jreused.md:
```diff
@@ -0,0 +1,651 @@
+ * Reference: https://gist.github.com/floralvikings/10290131
+ */
+public class AutoCompleteTextField extends TextField
+{	
```
Coding standard violation

damithc commented on collated/main/A0147995H.md:
```diff
@@ -0,0 +1,657 @@
+		return null;
+	}
+	
+	public EditCommand(int targetIndex, String taskName, Set<String> tags, Date startDate, Date endDate, RecurringType recurringType) throws IllegalValueException {
```
Line too long?

damithc commented on collated/main/A0147995H.md:
```diff
@@ -0,0 +1,657 @@
+                    FindCommand.MESSAGE_USAGE));
+        }
+        return new FindCommand(keywordSet, startTime, endTime, deadline, tagSet);
+    }
```
This method has very bad SLAP :-p

damithc commented on collated/test/A0135782Y.md:
```diff
@@ -0,0 +1,533 @@
+        // execute command and verify result
+        assertCommandBehavior(helper.generateAddCommand(toBeAdded),
+                String.format(AddNonFloatingCommand.MESSAGE_SUCCESS, toBeAdded),
+                expectedAB,
```
This variable name needs to be updated.

damithc commented on collated/test/A0135782Y.md:
```diff
@@ -0,0 +1,533 @@
+                String.format(AddNonFloatingCommand.MESSAGE_SUCCESS, toAdd),
+                expectedTM,
+                expectedComponentList);
+    }
```
Test methods seem to be very similar. Try to extract out similar parts into utility methods.

damithc commented on collated/test/A0135782Y.md:
```diff
@@ -0,0 +1,533 @@
+    }
+    
+    @Test
+    public void set_null_tasklist_throwAssert() {
```
Refer the test method naming convention again. The name should have no more than three parts.

damithc commented on collated/test/A0135782Y.md:
```diff
@@ -0,0 +1,533 @@
+            recurringManager.correctAddingOverdueTasks(null);
+            fail();
+        } catch (AssertionError ae) {
+            assertTrue(true);
```
JUnit has a better way to test for expected exceptions.

damithc commented on collated/test/A0147967J.md:
```diff
@@ -0,0 +1,1294 @@
+    		logic.execute(helper.generateAddCommand(t));
+    	}
+    	for(int i = 0; i < 3; i++)
+    		logic.execute("u");
```
Coding standard violation

damithc reviewed (commented):
Comments added to collated code.

brandonyeoxg committed:
Refactor RecurringTaskManager to include a RecurringTaskUtil class.

brandonyeoxg committed:
Attempt to increase test coverage.

brandonyeoxg committed:
Merge pull request #148 from CS2103AUG2016-W09-C2/fix-code-quality

Fix code quality

brandonyeoxg committed:
Fix annoying bug where recurring tasks gets appended for no reason.

Yichen-D committed:
Fix indentation

Yichen-D committed:
indentation

Yichen-D committed:
remove redundant catch

Yichen-D committed:
Merge pull request #151 from CS2103AUG2016-W09-C2/code-quality

Fix indentation

Yichen-D committed:
Remove redundant code

Yichen-D committed:
remove useless feature

billstark committed:
refactor edit/find parser

brandonyeoxg committed:
Add coverage test to increase coverage.

billstark committed:
Merge remote-tracking branch 'origin/development' into find-parser-refactor

# Conflicts:
#	src/main/java/seedu/address/logic/parser/Parser.java

brandonyeoxg committed:
Merge branch 'development' into fix-code-quality

# Conflicts:
#	src/main/java/seedu/address/model/ModelManager.java

Yichen-D committed:
Merge pull request #152 from CS2103AUG2016-W09-C2/code-quality

Remove redundant code

Yichen-D committed:
Merge pull request #153 from CS2103AUG2016-W09-C2/find-parser-refactor

Find parser refactor

Yichen-D committed:
Merge pull request #154 from CS2103AUG2016-W09-C2/fix-code-quality

Fix annoying bug where recurring tasks gets appended for no reason.

Yichen-D committed:
Fix minor bug, adjust layout

Yichen-D committed:
Fix bug

Yichen-D committed:
Merge pull request #155 from CS2103AUG2016-W09-C2/improvements

Fix minor bug, adjust layout

brandonyeoxg committed:
Fix bug on block command not being able to be added.

Yichen-D committed:
Add files via upload

brandonyeoxg committed:
Update tasklist to be used for CS2101 presentation.

Yichen-D committed:
Fix bugs

Yichen-D committed:
Merge pull request #156 from CS2103AUG2016-W09-C2/fix-many-bugs

Fix bugs

brandonyeoxg committed:
Fix bug to allow for add task from 7 nov 3pm to 4pm t/tag.

brandonyeoxg committed:
Merge pull request #157 from CS2103AUG2016-W09-C2/improve-useablity

Fix bug to allow for add task from 7 nov 3pm to 4pm t/tag.

brandonyeoxg committed:
Refactor methods in Parser that can be into RecurringDateParser

brandonyeoxg committed:
Add unit test for UniqueTaskListTest.java.

brandonyeoxg committed:
Add integration test into UniqueTaskListTest.java

brandonyeoxg committed:
Merge pull request #159 from CS2103AUG2016-W09-C2/add-unit-test

Add unit test for UniqueTaskListTest.java.

brandonyeoxg committed:
Merge pull request #160 from CS2103AUG2016-W09-C2/improve-useablity

Refactor methods in Parser that can be into RecurringDateParser

brandonyeoxg committed:
Merge pull request #161 from CS2103AUG2016-W09-C2/integration-test

Add integration test into UniqueTaskListTest.java

brandonyeoxg committed:
Remove un-needed Task constructor.
Refactor test in LogicManagerTest.

brandonyeoxg committed:
Merge pull request #162 from CS2103AUG2016-W09-C2/improve-code-coverage-quality

Remove un-needed Task constructor.

brandonyeoxg committed:
Add my own improvements.

brandonyeoxg committed:
Add new improvements to useability?

brandonyeoxg committed:
Update useability improvement hopefully.

brandonyeoxg committed:
Comment out LogicManagerTest line 884-885 for quick push to test new dog food.

brandonyeoxg committed:
Add useability, test will fail.

Yichen-D committed:
Fix testcases, change behaviour a bit.

brandonyeoxg committed:
Merge pull request #163 from CS2103AUG2016-W09-C2/improve-code-coverage-quality-patch

Fix testcases, change behaviour a bit.

Yichen-D committed:
Minor changes

brandonyeoxg committed:
Refactor methods related to RecurringTaskManager.

brandonyeoxg committed:
Merge branch 'improve-code-coverage-quality' of https://github.com/CS2103AUG2016-W09-C2/main into improve-code-coverage-quality

brandonyeoxg committed:
Refactor RecurringTaskManagerTest.java

brandonyeoxg committed:
Refactor TaskTester, try to add unit test for the task testing.

brandonyeoxg committed:
Remove exceptions that are no longer needed.

Yichen-D committed:
Merge pull request #164 from CS2103AUG2016-W09-C2/improve-code-coverage-quality

Changing of behavior of Task Master

brandonyeoxg committed:
Refactor RecurringTaskManager and also remove unused methods.

antonong94 committed:
Debug jar file not showing images in help

antonong94 committed:
Merge pull request #165 from CS2103AUG2016-W09-C2/edit_help_bug

Debug jar file not showing images in help

brandonyeoxg committed:
Fix failing test.

brandonyeoxg committed:
Merge pull request #167 from CS2103AUG2016-W09-C2/improve-code-coverage-quality

Refactor RecurringTaskManager and also remove unused methods.

brandonyeoxg committed:
Update architecture png.
Update Developer Guide to include activity diagram.

brandonyeoxg committed:
Update documentation

brandonyeoxg committed:
Update Logic.png

brandonyeoxg committed:
Edit activity diagram.

brandonyeoxg committed:
Merge pull request #170 from CS2103AUG2016-W09-C2/document-developer-guide

Document developer guide

brandonyeoxg committed:
Merge pull request #169 from CS2103AUG2016-W09-C2/development

Update master to development for v0.5rc

Yichen-D committed:
Update README.md

Yichen-D committed:
Change name of app in readme.md

brandonyeoxg committed:
Add an attempt on oom diagram

Yichen-D committed:
Change gui layout, delete useless stylesheet

Yichen-D committed:
Update MainWindow.java

Yichen-D committed:
fix edit bug

Yichen-D committed:
Correct edit behaviour

Yichen-D committed:
Merge pull request #174 from CS2103AUG2016-W09-C2/Adjustments

Adjustments

brandonyeoxg committed:
Change behaviour of recurring dates, now appends the next date without having to wait for the next recurring period to elapse.

brandonyeoxg committed:
Fix test cases to pass.

brandonyeoxg committed:
Add recurring period.

brandonyeoxg committed:
Fix test cases to pass.

brandonyeoxg committed:
Change behaviour of adding recurring tasks that are before the current date.

brandonyeoxg committed:
Fix failing test.

Yichen-D committed:
Fix agenda display,
add tentative gui tests for it
code quality not improved yet

Yichen-D committed:
Merge pull request #177 from CS2103AUG2016-W09-C2/improving-recurring-dates-patch

Fix agenda display,

brandonyeoxg committed:
Add new test for recurring manager.
Solve bug where archived tasks can still be archived.

brandonyeoxg committed:
Merge pull request #178 from CS2103AUG2016-W09-C2/improve-recurring-dates

Fix archive bug add 1 coverage test.

brandonyeoxg committed:
Improve code quality. Fix bug relating to recurring tasks.

brandonyeoxg committed:
Refactor to improve code quality.

brandonyeoxg committed:
Refactor recurring task manager and its utility class.

brandonyeoxg committed:
Anothr round of major refactoring.

brandonyeoxg committed:
Merge pull request #179 from CS2103AUG2016-W09-C2/improve-recurring-dates

Major refactoring and adding of collate tags

brandonyeoxg committed:
Merge pull request #180 from CS2103AUG2016-W09-C2/improve-recurring-dates

Major refactoring and adding of collate tags

brandonyeoxg committed:
Add user guide documentation.

brandonyeoxg committed:
Add user guide documentation.

Yichen-D committed:
first round of refactoring

Yichen-D committed:
Refactor second round, change output

brandonyeoxg committed:
Update developer guide diagram

brandonyeoxg committed:
Update developer guide diagram

brandonyeoxg committed:
Update non floating command description.

brandonyeoxg committed:
Update non floating command description.

Yichen-D committed:
Spot a bug? in edit with repeating tasks. Here is my solution to that.

brandonyeoxg committed:
Refactor task tester.

brandonyeoxg committed:
Refactor task tester.

Yichen-D committed:
Minor change on block command parser to add flexibility and minor changes fro taskcard

brandonyeoxg committed:
Add resources for developer guide.

brandonyeoxg committed:
Add resources for developer guide.

brandonyeoxg committed:
fix sequence diagram.

brandonyeoxg committed:
fix sequence diagram.

brandonyeoxg committed:
Edit developer guide.

brandonyeoxg committed:
Edit developer guide.

brandonyeoxg committed:
Change heading for testing section.

brandonyeoxg committed:
Change heading for testing section.

brandonyeoxg committed:
Refine testing section.

brandonyeoxg committed:
Refine testing section.

brandonyeoxg committed:
Edit testing section's integration example.

brandonyeoxg committed:
Edit testing section's integration example.

brandonyeoxg committed:
Add relevant images.

brandonyeoxg committed:
Add relevant images.

brandonyeoxg committed:
Merge pull request #183 from CS2103AUG2016-W09-C2/edit-documentation

Edit documentation

brandonyeoxg committed:
no message

brandonyeoxg committed:
Merge pull request #184 from CS2103AUG2016-W09-C2/development

update master to the latest version

brandonyeoxg committed:
Updated collated parts.

billstark committed:
change naming of method

billstark committed:
indentation

billstark committed:
naming issue

billstark committed:
naming issue

billstark committed:
display today bug fix

billstark committed:
Merge remote-tracking branch 'origin/development' into fix

# Conflicts:
#	src/main/java/seedu/address/logic/commands/EditCommand.java
#	src/main/java/seedu/address/model/task/TaskOccurrence.java

Yichen-D committed:
Merge pull request #182 from CS2103AUG2016-W09-C2/fix

first round of refactoring

Yichen-D committed:
Fix minor bug in edit, first update on sample data and test script, update collate

Yichen-D committed:
Update TestScript.md

Yichen-D committed:
Update TestScript.md

Yichen-D committed:
Merge pull request #185 from CS2103AUG2016-W09-C2/add-sampledata-and-testscript

Fix minor bug in edit, first update on sample data and test script, u…

Yichen-D committed:
no message

Yichen-D committed:
Merge pull request #186 from CS2103AUG2016-W09-C2/development

Development

Yichen-D committed:
Update collated file, change config setting to fit test

antonong94 committed:
Update UserGuide.md

Edit syntax

antonong94 committed:
Remove chinese character in Ui

antonong94 committed:
Merge pull request #187 from CS2103AUG2016-W09-C2/master1

Remove chinese character in Ui

antonong94 committed:
Update README.md

remove chinese character on agenda

antonong94 committed:
Update UserGuide.md

remove chinese characters on agenda

brandonyeoxg committed:
Minor refactoring to improve code quality based on coding standards.

brandonyeoxg committed:
Refactor task cases.

brandonyeoxg committed:
Refactor in more test cases.

brandonyeoxg committed:
Update collated file

brandonyeoxg committed:
increase coverage

brandonyeoxg committed:
Merge pull request #188 from CS2103AUG2016-W09-C2/refactor-patch

Refactor minor coding standard problems.

brandonyeoxg committed:
Update collate files.

brandonyeoxg committed:
Minor change.

brandonyeoxg committed:
Change app task title.

antonong94 committed:
Update AboutUs

brandonyeoxg committed:
update user guide.

brandonyeoxg committed:
Merge branch 'master' of https://github.com/CS2103AUG2016-W09-C2/main

brandonyeoxg committed:
update user guide.

brandonyeoxg committed:
refactor all packages. ensure that test passed.

Yichen-D committed:
Update TestScript.md

Yichen-D committed:
Minor change

brandonyeoxg committed:
update collate files.

Yichen-D committed:
Minor change

Yichen-D committed:
Change name of test folder
